src/service/gnuradio/examples/picoscope4000a-streaming.grc

```diff
@@ -1,15 +1,15 @@
 blocks:
-  - name: fair::picoscope::Picoscope4000a<float>
+  - name: picoscope
     id: fair::picoscope::Picoscope4000a
     template_args: "float,fair::picoscope::AcquisitionMode::Streaming"
     parameters:
       acquisition_mode: Streaming
```

Comment:
I didn't follow the recent picoscope changes by Magnus and you closely, but why is the acquisition mode now passed both as a template parameter and a setting? (curious, not changed in the PR)

Reply:
To me it looks like the setting is obsoleted by the template parameter and could be removed. I think for the BlockMode acquisition we also stilll have to make some changes in gr-digitizer where we could follow this up.